flight_planning/v1/telemetry.yaml

```diff
@@ -0,0 +1,250 @@
+info:
+  title: >-
+    Telemetry definitions used in flight planning interface.
+components:
+  schemas:
+    FlightTelemetry:
+      description: >-
+        When this information is present and the USS supports telemetry ingestion for a flight, the USS receiving this information should enqueue these telemetry reports to be delivered to the system, as if they were coming from the user, at (or soon after) the specified times.
+      type: object
+      properties:
+        states:
+          description: >-
+            The set of telemetry data that should be injected into the system (as if reported by the user or the user's system) at the appropriate times (and not before) for this flight.
+          type: array
+          items:
+            $ref: '#/components/schemas/AircraftState'
+          default: []
+    AircraftState:
+      description: State of an aircraft for the purposes of simulating the execution of a flight plan.
+      required:
+        - id
+        - timestamp
+      type: object
+      properties:
+        id:
+          description: >-
+            Unique identifier for this aircraft state/telemetry.  The content of an AircraftState with a given ID must not change over time.  Therefore, if a USS has already queued an aircraft state with this ID as telemetry to be delivered, this state may be ignored.
+          type: string
+        timestamp:
+          description: >-
+            Time at which this state is valid.  This is equivalent to the time of sensor measurement, so the USS's primary system under test should not be aware of this state until after this time.
+          anyOf:
+            - $ref: 'geotemporal.yaml#/components/schemas/Time'
+        timestamp_accuracy:
```

Comment:
This field (and a few others below) have defaults set to 0.0. Does that actually make sense? Shouldn't the default be a lack of value rather than 0? (i.e. no default)

Reply:
Since exact measurement of the physical world is impossible, I think true 0 can be reasonably considered a sentinel value.  The specific problem with defaults is that certain API implementation mechanisms (Google protocol buffers + gRPC) have a lot of trouble with default values that are not the default values of the data type (0 for numbers).  So, to be maximally compatible, we want all default values to be the default values of the data types unless there's an extremely good reason not to.

Reply:
Didn't think of that, makes sense!